README.md

```diff
@@ -29,7 +29,22 @@ Or install it yourself as:
     $ gem install peanut_labs
 
 ## Usage
-Follow the code, Luke (c) 
+You need to provide you app_id and and app_key. 
+Perfect place to do that in rails app is config/initialize/peanutlabs.rb file
+
+```
+PeanutLabs::Credentials.id = <YOUR APP ID>
+PeanutLabs::Credentials.key = <YOUR KEY>
+```
+
+You would need callback in your app to receive response from peanutlabs.
+To do that securely, you need to whitelist your route in rails routes.rb file, like this:
+
+```
+require 'peanut_labs/whitelist'
```

Comment:
It would be nice to have a short explanation about the source from where these hardcoded IPs come to whitelist.rb